docker/kubectl-bats/Dockerfile

```
@@ -23,12 +23,14 @@ USER root
 
 # setup bats
 RUN apt-get update && \
-    apt-get install -y curl git vim python3 python3-pip && \
+    apt-get install -y curl git vim python3 python3-pip jq && \
     git clone https://github.com/bats-core/bats-core.git "${BATS_HOME}"/bats-core && \
     git clone https://github.com/bats-core/bats-support.git "${BATS_HOME}"/test_helper/bats-support && \
     git clone https://github.com/bats-core/bats-assert.git "${BATS_HOME}"/test_helper/bats-assert && \
-    pip3 install -U niet && \
-    curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+    curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash && \
+    curl -L https://github.com/mikefarah/yq/releases/download/v4.34.2/yq_linux_amd64 -o /usr/bin/yq && \
+    chmod +x /usr/bin/yq && \
+    helm version && yq --version
```

If we are going to print the Helm and YQ versions during container image build, then we should do so in a separate `RUN` step right as the last `RUN` statement in the file.

```suggestion
    chmod +x /usr/bin/yq
```